index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
- [ ] [REQUIRED] I think it would be better if you add a more descriptive `<title>` to the document 📔 for the page to Be SEO friendly 🌐 & to help Assistive devices ♿.